### PR TITLE
update, adding new code

### PR DESCRIPTION
**Pull request by shajoezhu**

### CODE REVIEW

File: R/assert.R [hunk 1]
```diff
@@ -8,11 +8,19 @@ assert_is_character_scalar <- function(x) {
   }
 }
 
+assert_is_valid_version_label <- function(x) {
+  if (!(x %in% c("DRAFT", "APPROVED") || is.null(x))) {
+    abort("Version label must be 'DRAFT', 'APPROVED' or `NULL` but is '", x, "'.")
```
Comment (Melkiades) [inline R/assert.R:13]: not stop?

File: R/t_ds_slide.R [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    ) %>%
+    summarize_row_groups(label_fstr = "Discontinued the study") %>%
+    analyze_vars(
+      "DCSREAS",
+      .stats = "count_fraction",
+      denom = "N_col"
+    )
+
+  result <- lyt_to_side_by_side(lyt, adsl1, side_by_side)
+  result@main_title <- "Discontinue table"
```
Comment (Melkiades) [inline R/t_ds_slide.R:78]: better to use the accessor or build_table(main_title=)

Comment (edelarua) [reply R/t_ds_slide.R:78]: @Melkiades I think you meant to write `basic_table(main_title=)` :)

File: R/assert.R (hunk 1 above)
Comment (shajoezhu) [reply R/assert.R:13]: not a hard stop, as it will break the workflow

**Review by edelarua — APPROVED** [review]
Good to go! We can address the old table code in a different issue :)

**Pull request merged**